.buildkite/pipeline.yml

```diff
@@ -107,6 +107,26 @@ steps:
         artifact_paths:
           - "**/build/test-results/merged-test-results.xml"
 
+      - label: "🔬 Unit test FluxC"
```

Comment:
Minor (🔍): For consistency purposes, this could be renamed to a capital `Test` instead, just to match the rest of them.

PS: Same applies to `Unit test Login` below.

Reply:
👍 [c79408b](https://github.com/wordpress-mobile/WordPress-Android/pull/21261/commits/c79408bb3eba2e877a201c06d5083b079b687e8f)